crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1641,15 +1641,11 @@ impl CumulocityConverter {
                     .operations_for_device_mut(target)
                     .expect("entity should've been checked before that it's not a service");
 
-                let prev_operation = current_operations.remove_operation(&operation.name);
+                let prev_operation = current_operations.insert_operation(operation);
```

Comment:
This change seems weird at first ... but is correct!

Previously any operation with the same name was removed before adding a new version of the same operation.